dio/lib/src/sentry_dio_extension.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:dio/dio.dart';
+import 'package:sentry/sentry.dart';
+import 'sentry_transformer.dart';
+import 'sentry_client_adapter.dart';
+
+/// Extension to add performance tracing for [Dio]
+extension SentryDioExtension on Dio {
+  /// Adds support for automatic spans for http requests,
+  /// as well as request and response transformations.
+  /// This must be the last initialization step of the [Dio] setup, otherwise
+  /// your configuration of Dio might overwrite the Sentry configuration.
+  void addSentry({
+    bool recordBreadcrumbs = true,
+    bool networkTracing = true,
+    MaxRequestBodySize maxRequestBodySize = MaxRequestBodySize.never,
```

Comment:
isn't `small` the default value?

Reply:
Not sure about that, but I guess not as I've copied it from the HTTP integration. I'll double check it though.

Reply:
its `never`, my bad